_tour/traits.md

```diff
@@ -29,7 +29,7 @@ trait Iterator[A] {
 }
 ```
 
-Extending the `trait Iterator[A]` requires a type `A` and implementations of the methods `hasNext` and `next`.
+Extending the `trait Iterator[A]` requires a type `A`, and implementations of the methods `hasNext` and `next`.
```

Comment:
adding a comma here is incorrect. a comma would only be correct here if it introduced an independent clause

if you feel the sentence is hard to read without the comma, we need to find some other solution.

Reply:
No problem, maybe it just my opinion 👍🏻. 
I close this PR

Reply:
but I liked the other two changes!

Reply:
@SethTisue I'll remove the conflicting change and leave the other

Reply:
@SethTisue this is the new PR[#1665](https://github.com/scala/docs.scala-lang/pull/1665)